docs/usage.md

```diff
@@ -108,6 +108,45 @@ it should infer the parser based on the filename:
 osv-scanner --lockfile ':/path/to/my:projects/package-lock.json'
 ```
 
+### Custom Lockfiles
+
+If you have a custom lockfile that we do not support or prefer to do your own custom parsing, you can extract the custom lockfile information and create a custom intermediate file containing dependency information so that osv-scanner can still check for vulnerabilities. 
+
+Once you extracted your own dependency information, place it in a `osv-scanner-results.json` file, with the same format as the JSON output of osv-scanner, e.g.:
+
+```
+{
+  "results": [
+    {
+      "packageSource": {
+        "path": "/absolute/path/to/custom.lockfile",
+        "type": "lockfile"
```

Comment:
Also: this this value matter? Can we just omit this to make things simpler for users?

Reply:
Removed